intercom/write.go

```diff
@@ -0,0 +1,68 @@
+package intercom
+
+import (
+	"context"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/jsonquery"
+	"github.com/spyzhov/ajson"
+)
+
+type writeMethod func(context.Context, string, any) (*common.JSONHTTPResponse, error)
```

Comment:
Can you actually add `writeMethod` to `common/types/go` now that is used in several connectors now.

Reply:
As a tidy task I could apply some common methods or those coming from packages like jsonquery, urlbuilder, paramsbuilder. But that kind of refactoring can only happen after adding mock tests for all connectors. Let me know if I could do such task.